pkg/backends/otlp/backend.go

```diff
@@ -280,27 +283,32 @@ func (bd *Backend) SendMetricsAsync(ctx context.Context, mm *gostatsd.MetricMap,
 			if len(t.Histogram) != 0 {
 				opts = append(opts, data.WithHistogramDataPointCumulativeBucketValues(t.Histogram))
 			}
-			group.insert(bd.is, resources, data.NewMetric(name).SetHistogram(
+			currentGroup.insert(bd.is, resources, data.NewMetric(name).SetHistogram(
 				data.NewHistogram(data.NewHistogramDataPoint(uint64(t.Timestamp), opts...)),
 			))
 		}
 	})
 
-	var errs error
-	for _, b := range group.batches {
+	eg, ectx := errgroup.WithContext(ctx)
+	for _, b := range currentGroup.batches {
 		atomic.AddUint64(&bd.batchesCreated, 1)
-		err := bd.postMetrics(ctx, b)
-		if err != nil {
-			bd.logger.WithError(err).WithFields(logrus.Fields{
-				"endpoint": bd.metricsEndpoint,
-			}).Error("Issues trying to submit data")
-			errs = multierr.Append(errs, err)
-		} else {
-			atomic.AddUint64(&bd.batchesSent, 1)
-			atomic.AddUint64(&bd.seriesSent, uint64(b.lenMetrics()))
-		}
+		func(g group) {
+			eg.Go(func() error {
```

Comment:
Cool this means async sends? Do we have any retries in the otlp backend?

Reply:
yeah retry is in the `postMetrics`